files/jquery.inputmask/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "jquery.inputmask",
+  "repo": "RobinHerbots/jquery.inputmask",
+  "files": {
+    "basePath": "dist/min/inputmask/",
+    "include": ["jquery.inputmask.min.js"]
```

Comment:
Yes, in this case you can delete the include parameter completely. We want everything in that folder